docs/installation.md

```diff
@@ -19,40 +19,34 @@
     singularity build r2dt docker://rnacentral/r2dt
     ```
 
-2. Setup a precomputed data library _(171 MB, last updated Aug 23, 2024)_:
+2. Enter an interactive terminal session:
     ```bash
-    curl -O -L https://github.com/r2dt-bio/R2DT/releases/download/v2.0/cms.tar.gz
-    tar -xzf cms.tar.gz
-    export R2DT_LIBRARY=<path to precomputed library>
-    ```
-
-3. Verify that the installation worked:
-    ```bash
-    docker run --entrypoint r2dt.py rnacentral/r2dt draw --help
-    ```
-
-    Or in Singularity:
-    ```bash
-    singularity exec r2dt r2dt.py draw --help
-    ```
-
-4. Enter an interactive terminal session:
-    ```bash
-    docker run -it -v $R2DT_LIBRARY:/rna/r2dt/data/cms -v `pwd`:/rna/r2dt/temp rnacentral/r2dt
+    docker run -it -v `pwd`:/rna/r2dt/temp rnacentral/r2dt
     ```
 
     - `-it` - start an interactive session
-    - `-v $R2DT_LIBRARY:/rna/r2dt/data/cms` - mount the precomputed data library folder as `/rna/r2dt/data/cms` inside the container. ⚠️ Note that `<path_to_cms>` should be a full path.
     - make the current working directory available inside the container as `/rna/r2dt/temp`:
         ```bash
         -v `pwd`:/rna/r2dt/temp
         ```
 
     Any file placed in `/rna/r2dt/temp` within the container will be available on the host machine after the Docker container exits. The current directory is mounted inside the container so that all code and data changes are instantly reflected in the container.
 
+:::{note}
+
+Starting with version 2.2, downloading a precomputed library is no longer necessary. For older versions, however, you must download the library manually and mount it inside the container using Docker’s `-v` option. For example:
+
+```bash
+curl -O -L https://github.com/r2dt-bio/R2DT/releases/download/v2.0/cms.tar.gz
+tar -xzf cms.tar.gz
+export R2DT_LIBRARY=<path to precomputed library>
+docker run -it -v $R2DT_LIBRARY:/rna/r2dt/data/cms -v `pwd`:/rna/r2dt/temp rnacentral/r2dt
+```
+:::
+
 ## Setup a development environment
 
-Set up a precomputed library as described above and run the container using [just](https://just.systems):
+To run a development container using [just](https://just.systems), first set up the precomputed library as described above, then:
```

Comment:
> first set up the precomputed library as described above

Is it necessary though?

Reply:
Good catch, thank you! Will fix it in a bit.